pkgs/development/python-modules/poetry-dynamic-versioning/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, pythonOlder
+, dunamai
+, jinja2
+, markupsafe
+, poetry-core
+, pytest
+, tomlkit
+}:
+buildPythonPackage rec {
+  pname = "poetry-dynamic-versioning";
+  version = "0.14.1";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    repo = pname;
+    owner = "mtkennerly";
```

Comment:
I wouldn't mind if we switch repo and owner to muscle memory.

Reply:
Done.